CHANGELOG.md

```diff
@@ -5,11 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [v0.2.1] Unrealeased
+## [v0.2.1] Unreleased
```

Comment:
Why not make a release?

Reply:
I was planning on doing it for https://github.com/giantswarm/prometheus-operator-app/pull/23 :D